### PR TITLE
Ml project 4.3

### PR DESCRIPTION
**Pull request by priyanka-TL**

### CODE REVIEW

File: apis/ml-projects/ml-projectsapi.yaml [hunk 1]
```diff
@@ -0,0 +1,1863 @@
...
+- url: https://staging.open-sunbird.org/api
+  variables: {}
+paths: 
+  /userProjects/mlprojects/v1/details/{projectId}:
+    get:
+      tags:
+      - userProjects APIs
+      summary: Get Project Details.
+      description: |-
+        This API is used to get the Project Details. This api will return the project details based on the role and location Id.
```
Comment (mohitga) [inline apis/ml-projects/ml-projectsapi.yaml:23]: @priyanka-TL where are the role and location id provided?

File: apis/ml-projects/ml-projectsapi.yaml [hunk 2]
```diff
@@ -0,0 +1,1863 @@
...
+      deprecated: false
+      security:
+        - bearer: []
+  /userProjects/mlprojects/v1/add:
+    post:
+      tags:
+      - userProjects APIs
+      summary: Add project.
+      description: |-
+        This API is associated with creating project on the Sunbird Platform.
```
Comment (mohitga) [inline apis/ml-projects/ml-projectsapi.yaml:80]: @priyanka-TL  is this the right definition of this API "This API creates projects on top user object"?

File: apis/ml-projects/ml-projectsapi.yaml [hunk 3]
```diff
@@ -0,0 +1,1863 @@
...
+                      type: string
+                      example: projectId/userId/imageName
+              remarks:
+                description: Represents the notes for task
+                type: string
+                example: Tasks completed
+              assignee: 
+                description: Represents who is the assignee of this task
+                type: string
+                example: Aman
```
Comment (mohitga) [inline apis/ml-projects/ml-projectsapi.yaml:1047]: Is Assignee name (string) or user id?

File: apis/ml-projects/ml-projectsapi.yaml [hunk 4]
```diff
@@ -0,0 +1,1863 @@
...
+                      type: string
+                      example: Task 2
+                    description: 
+                      description: Represents the description of task
+                      type: string
+                      example: Task 2 description
+                    status: 
+                      description: Represents the status of task
+                      type: string
+                      example: notStarted
```
Comment (mohitga) [inline apis/ml-projects/ml-projectsapi.yaml:1069]: Add more example of status

File: apis/ml-projects/ml-projectsapi.yaml [hunk 5]
```diff
@@ -0,0 +1,1863 @@
...
+              label: 
+                type: string
+                example: teacher
+        status: 
+          type: string
+          example: notStarted/inProgress/completed
+        lastDownloadedAt: 
+          type: string
+          example: 2020-09-29T09:08:41.667Z
+        payload:
```
Comment (mohitga) [inline apis/ml-projects/ml-projectsapi.yaml:1130]: please add description to the payload, What payload is this etc

File: apis/ml-projects/ml-projectsapi.yaml [hunk 6]
```diff
@@ -0,0 +1,1863 @@
...
+      deprecated: false
+      security:
+        - bearer: []
+  /userProjects/mlprojects/v1/sync/{projectId}:
+    post:
+      tags:
+      - userProjects APIs
+      summary: Sync project.
+      description: |-
+        This API is used to sync the project details.
```
Comment (mohitga) [inline apis/ml-projects/ml-projectsapi.yaml:144]: @priyanka-TL  what does sync project sync means? what are the entity /databases across project is synced?

**Review by mohitga — CHANGES_REQUESTED** [review]

File: apis/ml-projects/ml-projectsapi.yaml (hunk 3 above)
Comment (priyanka-TL) [reply apis/ml-projects/ml-projectsapi.yaml:1047]: its assignee name

**Comment by mohitga** [comment]
@priyanka-TL  this branch is showing conflicts after your changes, please check

**Pull request merged**